config/openshift/base/500-webhooks.yaml

```diff
@@ -47,3 +47,9 @@ webhooks:
       name: tekton-operator-webhook
       namespace: openshift-operators
   name: config.webhook.operator.tekton.dev
+  objectSelector:
+    matchExpressions:
+      - key: operator.tekton.dev/release
+        operator: Exists
+    #matchLabels:
+    #  app.kubernetes.io/part-of: tekton-operator
```

Comment:
We could do one or the other, hence me adding a `app.kubernetes.io/part-of` in the tekton-config-default.yml as an example.

Reply:
Note, we could have a very specific label for this if we want to.

Reply:
👍